Homework 3/script.js

```diff
@@ -18,6 +18,18 @@ $( document ).ready(function() {
     }
 	});
 
+	$("#createProductButton").click(function(){
+		if($("#productNameInput").val() == "" || $("#productImageInput").val() == "" || $("#productCostInput").val() == ""){
+			$("#logInErrorMessage").append("* You must fill in all of the fields.");
+		}
+		else{
+			productManager.createProduct($("#productNameInput").val(), parseInt($("#productCostInput").val()), $("#productImageInput").val(), user._store);
+			$("#productNameInput").value='';
```

Comment:
Again there's some code duplication. If you abstract the above in line 22 into it's own array, you can reuse it. Actually, you can probably just refactor and use a class. Don't think `.value` works actually. That's vanilla js.

```js
$("#productNameInput, #productCostInput, #productImageInput").val("")
```

You can chain css selectors together.